Source/SwiftLintFramework/Models/SwiftVersion.swift

```diff
@@ -38,6 +39,13 @@ public extension SwiftVersion {
             }
         }
 
+        if !Request.disableSourceKit,
+            case let opaqueReturnFile = File(contents: "func a() -> some A {}"),
+            case let expectedKinds: [SyntaxKind] = [.keyword, .identifier, .keyword, .typeidentifier],
+            opaqueReturnFile.syntaxMap.tokens.compactMap({ SyntaxKind(rawValue: $0.type) }) == expectedKinds {
```

Comment:
this should eventually be the replaced with the [the new SourceKit request](https://github.com/apple/swift/pull/22010)

Reply:
Done in 0652b32